hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetrics.java

```diff
@@ -71,10 +71,13 @@ public class OMMetrics implements OmMetadataReaderMetrics {
   private @Metric MutableCounterLong numSnapshotCreates;
   private @Metric MutableCounterLong numSnapshotDeletes;
   private @Metric MutableCounterLong numSnapshotLists;
+  private @Metric MutableCounterLong numSnapshotRenames;
   private @Metric MutableCounterLong numSnapshotDiffJobs;
   private @Metric MutableCounterLong numSnapshotInfos;
   private @Metric MutableCounterLong numSnapshotPurges;
   private @Metric MutableCounterLong numSnapshotSetProperties;
+  private @Metric MutableCounterLong numCancelSnapshotDiffs;
+  private @Metric MutableCounterLong numListSnapshotDiffJobs;
```

Comment:
Thanks for the patch @jojochuang. Can you also add metrics to track [OMSnapshotMoveTableKeysRequest.java](https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotMoveTableKeysRequest.java) and [OMSnapshotMoveDeletedKeysRequest.java](https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotMoveDeletedKeysRequest.java)?

Reply:
@SaketaChalamchala good idea. This PR implements metrics/tests for client operations. We should add the same for internal operations like you mentioned.

Let me open another jira to track the internal snapshot operations.